Matveibos/hometask-2/get_middle_character.rb

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env ruby
+
+# frozen_string_literal: true
+
+def run_cli
+  loop do
+    puts 'Enter string'
+    string = gets.chomp
+    break if string == 'exit!'
+
+    if string.empty?
```

Comment:
It is completely fine to use if/else here, but check out how you can rewrite this statement with the ternary expression. You can use them, if it is not affecting readability for the code, and this is the case for this code, IMHO

Reply:
Do you mean combine break if string == 'exit!'  and if string.empty? ...  with the ternary expression?

Reply:
No, I mean that you can rewrite that to this:
```
string.empty? ? puts 'String can not be blank!' : puts "Middle character: #{get_middle(string)}"
```

Reply:
I did it like this :
if st.empty? ? (puts 'String can not be blank!') : (puts RESULT + get_middle(st))
# comment, but rubocop warns: Lint/EmptyConditionalBody: Avoid if branches without a body.
end